protobuf/acs12.proto

```diff
@@ -0,0 +1,34 @@
+/**
+ * AElf Standards ACS12(User Contract Transaction Fee Standard)
+ *
+ * Used to manage the transaction fee of user contract.
+ */
+syntax = "proto3";
+
+package acs12;
+
+import public "aelf/options.proto";
+import public "google/protobuf/empty.proto";
+import public "google/protobuf/wrappers.proto";
+import "aelf/core.proto";
+
+option (aelf.identity) = "acs12";
+option csharp_namespace = "AElf.Standards.ACS12";
+
+service UserContractMethodFeeProviderContract {
```

Comment:
`acs12` is not only used to provide MethodFee but also used to denote the UserContract (which requires CodeCheck). So is it better to call it `UserContract`?

Reply:
Done.